lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/download/DownloadListPackageRequest.json

```diff
@@ -9,6 +9,14 @@
 		"zipFileName": {
 			"description": "Optional parameter to set the name of the resulting zip file.",
 			"type": "string"
+		},
+		"includeManifest": {
+			"description": "When set to true, a metadata manifest files will be include in the package.  The manifest will include all metadata for each file in the package.",
```

Comment:
"When set to true, a metadata manifest files will be include in the package.  The manifest will include all metadata for each file in the package." 
->
"When set to true, a metadata manifest file will be included in the package.  The manifest will include all the metadata for each file in the package."

Reply:
Also what is the default value (or how null is interpreted).